src/components/Dapp.jsx

```diff
@@ -38,7 +38,7 @@ export default function Dapp() {
   async function handleFollow() {
```

Comment:
@6rivu there's a UX bug, which is that after sending the follow transaction, we want to clear the text inputs. I'll let you figure that out.